### PR TITLE
Remove extra horizontal line and spacing

### PR DESCRIPTION
**Pull request by tengqm**
This PR removes the annoying border line and extra spacing at the bottom on each page.

### CODE REVIEW

**Comment by tengqm** [comment]
What this PR does:

image

File: layouts/docs/baseof.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
             {{ if (and (not .Params.hide_feedback) (.Site.Params.ui.feedback.enable) (.Site.GoogleAnalytics)) }}
               {{ partial "feedback.html" .Site.Params.ui.feedback }}
             {{ end }}
-            <div class="text-muted mt-5 pt-3 border-top">{{ partial "page-meta-lastmod.html" . }}</div>
+            <div class="text-muted">{{ partial "page-meta-lastmod.html" . }}</div>
```
Comment (kbhawkey) [inline layouts/docs/baseof.html:28]: Hi @tengqm .
What do you think about removing the first `<div></div>`? The partial `page-meta-lastmod.html` from the theme includes this div element?

Comment (tengqm) [reply layouts/docs/baseof.html:28]: checking

Comment (tengqm) [reply layouts/docs/baseof.html]: @kbhawkey you are right. The partial already contains a DIV. An extra DIV is not needed. Fixed.

**Comment by sftim** [comment]
/area web-development
LGTM but I haven't looked thoroughly

**Comment by kbhawkey** [comment]
Page previews:
https://deploy-preview-28615--kubernetes-io-main-staging.netlify.app/docs/home/
https://deploy-preview-28615--kubernetes-io-main-staging.netlify.app/docs/concepts/
https://deploy-preview-28615--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/controllers/job/
https://deploy-preview-28615--kubernetes-io-main-staging.netlify.app/docs/reference/command-line-tools-reference/kube-proxy/

Current:
https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/

**Pull request merged**